Description: Fix search for init file (GNU Emacs 27.1).
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1010152

Patch from https://debbugs.gnu.org/cgi/bugreport.cgi?bug=42827#16

diff --git a/src/xrdb.c b/src/xrdb.c
index e3a1fcb15a..3d7f715c88 100644
--- a/src/xrdb.c
+++ b/src/xrdb.c
@@ -289,9 +289,9 @@ get_user_app (const char *class)
       /* Check in the home directory.  This is a bit of a hack; let's
 	 hope one's home directory doesn't contain ':' or '%'.  */
       char const *home = get_homedir ();
-      db = search_magic_path (home, class, "%L/%N");
+      db = search_magic_path (home, class, "/%L/%N");
       if (! db)
-	db = search_magic_path (home, class, "%N");
+	db = search_magic_path (home, class, "/%N");
     }
 
   return db;
